brillo-algorithms/brillo-algorithms.cabal

```diff
@@ -15,17 +15,17 @@ description:
 synopsis:
   Data structures and algorithms for working with 2D graphics.
 
+tested-with:   GHC ==9.2.8 || ==9.4.8 || ==9.6.6 || ==9.8.2 || ==9.10.1
```

Comment:
Only added GHC versions that support GHC2021 language extension which you're using as default-language everywhere..